mimir-knowledge/src/db/migrations/053_seed_connector_predicates.sql

```
@@ -0,0 +1,79 @@
+    ('shipped_by', 'Subject was shipped by an organization', FALSE),
+    ('delivered_to', 'Subject is delivered to a location', FALSE),
+    ('has_ticket', 'Subject has a ticket', FALSE),
+    ('issued_by', 'Subject was issued by an organization', FALSE)
+ON CONFLICT(name) DO UPDATE SET
+    description = excluded.description,
+    sensitive = excluded.sensitive;
+
+-- 2. Self-aliases for the new canonical predicates. UPSERT so a stale alias
+-- row is repointed to the canonical id instead of being silently kept.
+INSERT INTO relationship_type_aliases (alias, relationship_type_id)
+SELECT name, id FROM relationship_types WHERE name IN (
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)